docs/CHANGELOG.asciidoc

```diff
@@ -8,16 +8,2498 @@
+
+Lens & Visualizations::
+Adds the ability to duplicate layers in *Lens* {kibana-pull}140603[#140603]
+
+Machine Learning::
+* Notifications page {kibana-pull}140613[#140613]
+* Explain Log Rate Spikes: Add option to view grouped analysis results {kibana-pull}140464[#140464]
+* Stubs out UI for the ML Inference Pipeline panel {kibana-pull}140456[#140456]
+* Attach the anomaly charts embeddable to Case {kibana-pull}139628[#139628]
+* Log pattern analysis UI {kibana-pull}139005[#139005]
+* Attach the anomaly swim lane embeddable to Case {kibana-pull}138994[#138994]
+
+Management::
+* Updates the AggConfig for nested sibling pipeline aggs {kibana-pull}140650[#140650]
+* Adds the ability to allow variables in URL Drilldown titles {kibana-pull}140076[#140076]
+* Enables time series downsampling action in ILM configurations {kibana-pull}138748[#138748]
+* Adds the composite runtime field editor {kibana-pull}136954[#136954]
+
+Observability::
+Feat(slo): Create basic SLO route {kibana-pull}139490[#139490]
+
+Osquery::
+Adds Osquery results to Case {kibana-pull}139909[#139909]
+
+Security::
+Adds the ability to set a default Access Agreement for all `xpack.security`-level authentication providers {kibana-pull}139217[#139217]
+
+For more information about the features introduced in 8.5.0, refer to <<whats-new,What's new in 8.5>>.
+
+[[enhancements-and-bug-fixes-v8.5.0]]
+=== Enhancements and bug fixes
+
+For detailed information about the 8.5.0 release, review the enhancements and bug fixes.
+
+[float]
+[[enhancement-v8.5.0]]
+=== Enhancements
+Alerting::
+* 141189 alerts table performance {kibana-pull}141385[#141385]
+* 141119 remove visibility toogle + use_columns refactor {kibana-pull}141250[#141250]
+* Adds Stats on top of execution logs {kibana-pull}140883[#140883]
+* Adds the Logs tab to Rules and Connectors UI {kibana-pull}138852[#138852]
+* Adds "exclude previous hits" check box to ESQuery rule form {kibana-pull}138781[#138781]
+* The ES Query Rule Type now supports Runtime Mappings and the Fields parameters when using an Elasticsearch DSL query {kibana-pull}138427[#138427]
+
+APM::
+* Service metrics on inventory page {kibana-pull}140868[#140868]
+* Sort trace samples {kibana-pull}140589[#140589]
+* Setting for tail sampling storage limit in Fleet integration {kibana-pull}140567[#140567]
+* Adds AWS Lambda metrics to "Metrics" tab UI {kibana-pull}140550[#140550]
+* Adds Experimental Mode for APM UI {kibana-pull}139553[#139553]
+* Rename JVMs to Metrics {kibana-pull}138437[#138437]
+* Display partial data buckets differently and message in the chart tooltips {kibana-pull}137533[#137533]
+
+Dashboard::
+Adds the ability to view panel-level filters and queries {kibana-pull}136655[#136655]
+
+Discover::
+* Adds the ability to only show reset height button when current height is not the default {kibana-pull}143013[#143013]
+* Adds the ability to persist histogram height to local storage {kibana-pull}140997[#140997]
```

Comment:
I'd suggest to remove them, since those are fixes to #139446, so those bugs didn't escape into the wilderness FYI @davismcphee 
```suggestion
```